WolvenKit.App/ViewModels/Documents/DocumentViewModel.cs

```diff
@@ -34,7 +34,7 @@ public DocumentViewModel(string path) : base(Path.GetFileName(path), path)
     public bool IsDirty
     {
         get => _isDirty;
-        protected set => SetProperty(ref _isDirty, value);
+        set => SetProperty(ref _isDirty, value);
```

Comment:
Why did you make it public? Should kept private/protected.

Reply:
The compiler didn't let me and insisted it needed to be public. It's protected in the interface, though.

Reply:
Remove the setter from the interface

Reply:
Ah, yeah, makes sense. Fixed